### PR TITLE
Release cleanup

### PR DESCRIPTION
**Pull request by aliberts**
## What this does

- Update readme & cleanup obsolete stuff
- Improve example 1
- Improve PR template
- Fix `pretrained_policy_path` when using a pretrained policy from the hub
- Fix some typos

### CODE REVIEW

File: lerobot/common/datasets/lerobot_dataset.py [hunk 1]
```diff
@@ -62,14 +63,16 @@ def features(self) -> datasets.Features:
...
+        """Keys to access images from cameras."""
         image_keys = []
         for key, feats in self.hf_dataset.features.items():
             if isinstance(feats, datasets.Image):
                 image_keys.append(key)
         return image_keys + self.video_frame_keys
 
     @property
-    def video_frame_keys(self):
+    def video_frame_keys(self) -> list[str]:
```
[the remark this answers is not in the document]
Comment (michel-aractingi) [reply lerobot/common/datasets/lerobot_dataset.py:74]: Sorry to jump in here, but the datasets I tested with don't have the 'camera_keys' attribute. So I had to change it back to 'image_keys' to run the lerobot/scripts/visualize_dataset.py.

So probably 'camera_keys' should be added as an attribute in the datasets unless it's a problem on my side only.

[the remark this answers is not in the document]
Comment (Cadene) [reply lerobot/common/datasets/lerobot_dataset.py:74]: Are you on main? @michel-aractingi 

Could you provide a code snippet for minimal reproduction

[the remark this answers is not in the document]
Comment (michel-aractingi) [reply lerobot/common/datasets/lerobot_dataset.py:74]: sorry it was my fault. I had pulled the latest version of main but did not reinstall the library.  Sorry for the confusion.

[the remark this answers is not in the document]
Comment (Cadene) [reply lerobot/common/datasets/lerobot_dataset.py:74]: There might be something wrong on our side. You shouldnt have to reinstall the library ideally ^^
cc @aliberts

[the remark this answers is not in the document]
Comment (aliberts) [reply lerobot/common/datasets/lerobot_dataset.py:74]: @michel-aractingi Do you use poetry or pip?

[the remark this answers is not in the document]
Comment (michel-aractingi) [reply lerobot/common/datasets/lerobot_dataset.py:74]: I am using pip. I will try to revert back to previous pushes to see if I can retrace the steps and recreate the error.

**Pull request merged**